adoc/MAIN.release-notes.adoc

```diff
@@ -103,12 +107,39 @@ into a {tf} 0.12 state without tearing it down completely.
 
 [WARNING]
 =========
-When adding ``lifecycle { ignore_change = [user_data] }`` in your master and
+When adding `lifecycle { ignore_change = [user_data] }` in your master and
 worker instances, you will effectively prevent updates of nodes, should you or
-SUSE update the ``user_data``. This should be removed as soon as possible after the
+SUSE update the `user_data`. This should be removed as soon as possible after the
 migration to {tf} 0.12.
 =========
 
+[[vmware_precaution_nullstrings]]
+==== Extra precautions for automations based on `output.tf` on VMWare
+
+If you are running any automations on your VMWare deployment of {productname} that
+relies on the {tf} output as defined in `output.tf`, you might encounter some `null` values in the value slice after
+the update to {tf} 0.12. For example:
+
+----
+Outputs:
+ip_load_balancer = [
+  "10.84.73.8",
+]
+ip_masters = [
+  [
+    null,
+  ],
+]
+ip_workers = [
+  [
+    null,
+  ],
+]
+----
+
+You can work around this problem by joining another node to the cluster once {tf} 0.12 is installed.
```

Comment:
There is no need to do this. I verified this along with @atighineanu. Following what is described in https://github.com/SUSE/avant-garde/issues/1400#issue-582947104:

* Upgrade terraform to 0.12
* Run `terraform 0.12upgrade` (use `terraform 0.12upgrade -yes` if you want to skip interactive confirmation) on the folder containing your `tfstate` file
* Run `terraform apply`, without having to perform any manual changes on your terraform files

From now on, you can run `terraform refresh` and `terraform output` at will, everything is now properly synced because of the `terraform apply`.

Had you executed `terraform refresh` and then `terraform output` **without following this instructions** would have resulted in the `null` outputs.